//
//  SearchViewController.swift
//  SearchBar
//
//  Created by user143635 on 10/23/18.
//  Copyright © 2018 SHUBHAM AGARWAL. All rights reserved.
//

import UIKit

var mySovIndex = 0
var indexSovNumber = 0

var abstract_codes = ["","","","","","","","","","","O: 01",
                      "O: 02",
                      "O: 03",
                      "O: 04",
                      "O: 05",
                      "O: 06",
                      "O: 07",
                      "O: 08",
                      "P 01 ",
                      "P 02",
                      "P 03",
                      "P 04",
                      "P 05",
                      "P 06",
                      "P 07",
                      "P 08",
                      "P 09",
                      "P 10",
                      "P 11",
                      "P 12",
                      "P 13",
                      "P 14",
                      "P 15",
                      "P 16",
                      "P 17",
                      "P 18",
                      "P 19",
                      "P 20",
                      "P 21",
                      "P 22",
                      "P 23",
                      "P 24",
                      "P 25",
                      "P 26",
                      "P 27",
                      "P 28",
                      "P 29",
                      "P 30",
                      "P 31"]

var abstract_titles = ["Organizing Committee","Other Committees","Messages","Message from Dr. Mahatme","Message from ISHICON 2018 President","Message from ISHICON 2018 Secretary","Message from ISHICON 2018 Chairman",
                       "Message from ISHICON 2018 Vice Chairman","Message from ISHICON 2018 Organizing Secretary","Abstracts",
                       "Kidney allocation based upon virtual cross match in deceased donor program",
                        "Impact of Donor Specific anti- HLA Antibodies in Haploidentical HSCT",
                        "Standardization of HLA High resolution Typing by MiniSeq ",
                        "Successful Renal Transplant in HLA Sensitized Patients: Report of 15 cases from a tertiary healthcare center ",
                        "Clinical Relevance of anti-HLA antibodies in Renal Transplantation ",
                        "Pre transplant MICA antibodies and Impact on Renal allograft outcome",
                        "HLA matching at epitope level: An effective approach to provide platelet transfusion support in platelet refractory patients",
                        "HLA-KIR ligands associated with myelogenous leukemias in Indian Population: A case control study",
                        "Compelling Evidence for Multi-linguistic Indian National Registry",
                        " An Analysis of Co-relation between Luminex Lysate Crossmatch & Single Antigen Bead (SAB) Assay ",
                        " Old truly Gold? Revisiting old Norms: Implications of Isolated T-cell positive Flow-cytometric Crossmatches  ",
                        " IgG anti-HLA auto-antibody: A case report  ",
                        " Rituximab: Friend or Foe of flow cytometric crossmatch ",
                        " Donor Specific HlA DQ Antibodies may contribute to poor graft outcome after renal transplantation ",
                        "  Utility of C3d Testing for Detecting Clinically Relevant Antibodies –A Case Study ",
                        "  Red cell allo-antibody (Anti- M) resulting in false positive CDC Crossmatch",
                        "  HLA DSA Class II positivity in renal transplant – dangerous if ignored!!!",
                        "  Evaluation of electronic DP epitype profiling to determine DP Donor-Specific Antibodies during virtual crossmatch",
                        "  Predominance of HLA class II antibodies in serum of kidney allograft recipients having antibody mediated rejection",
                        "  Anti – HLA Antibody Detection – Technical Issues and Possible Solutions",
                        "  Effect of Anti-idiotypic antibodies on renal transplant and its outcome",
                        "  Allogenic – IgM antibodies detrimental / beneficial?",
                        "  HLA DQA1-DQB1 genotypes and haplotypes frequencies in patients with CeD from Indian population ",
                        "  Promoter sequence divergence of classical HLA class I genes ",
                        "  Relevance of HLA-B*57:01 screening for abacavir hypersensitivity in North Indians",
                        "  Ankylosing Spondylitis and HLA B*27",
                        "  HLA association with Systemic sclerosis in North Indian population    ",
                        "  Relevance of anti-angiotensin 1 receptor antibodies in acute renal allograft rejection ",
                        "  Role of MHC restricted T cells in aetiopathogenesis of Autoimmune disorders: Hypoparathyroidism and Vitiligo ",
                        "  Association between HLA-DRB1 and TNF-α polymorphism with the clinical and immunological characteristics of systemic lupus erythematosus form south Indian population ",
                        "  Awareness among parents of children with multi-transfused thalassemia major-A cross sectional study",
                        "  Importance of red cell serology in optimizing transfusion strategy of patient undergoing allogeneic hematopoietic stem cell transplant ",
                        "  Importance of Antibody profile in selecting donor for sensitized patients for better outcome after renal transplantation- IKDRC-Ahmedabad experience ",
                        " Case Reports: Advantage of high resolution HLA typing for donor selection for HSCT recipients  ",
                        " Impact of KIR receptor- ligand match/mismatch in post cyclophosphamide haploidentical transplant graft rejection ",
                        " Effect of Anti-HLA Antibodies in graft failure in Haploidentical Hematopoietic Stem Cell Transplantation. ",
                        " Post HSCT monitoring by molecular methods: an AIIMS experience  ",
                        " HSCT for thalassemia: Approaches for identifying family donors other than siblings  ",
                        " Quality Indicators in Tissue Typing in NABL Accrediated Laboratory  "]

var abstract_info = ["","","","","","","","","","",
    "Abhinav Seth, Ashish Sharma, Deepesh Kenwar Benjamine, Sarbpreet Singh, Ranjana Walker Minz, Gaurav Shankar Pandey, Vidyasagar Kallepalli \n\nPostgraduate Institute of Medical Research and Education, Sector 12, Chandigarh, India./n/nIntroduction: A negative CDC cross match is essential in cadaver renal transplants. Patients with high anti-HLA titers are often not listed in active transplant list due to a high probability of a positive CDC cross match and poorer outcomes. Detection of donor specific anti-HLA antibodies (DSA) on Luminex Single-Antigen Beads can improve short and long term outcomes.   \n\nMethods: All sensitized recipients were screened by Luminex for anti-HLA antibodies (n=63). Pretransplant HLA genotyping for A,B and DR was performed using Luminex/sequence specific primers at the time of donor identification. Patients with negative virtual crossmatch (DSA MFI less than 1000) were called for CDC X match. This method was used in eight sensitized cadaver kidney recipients (history of blood transfusion, previous transplant, multiple pregnancies or previous CDC positive and with antibody MFI more than 5000 on Luminex with peak MFI of 21444). \n\nResults: The mean age of recipients and donor was 39.38±9.43 and 38.13±8.31years respectively while M:F ratio was 2:6. The causes of ESRD were diabetic unknown (n=3) nephropathy (n=2), FSGS (n=1), IgA nephropathy (n=1), MPGN type I (n=1). The mean cold ischemia time was 473.86 minutes. The average follow up was 2.1±1.89 months. No patient had hyperacute rejection or infection and mean creatinine at discharge and one month was 1.27±0.24mg/dl and 1.13±0.3mg/dl. Biopsy proven acute rejection was seen in one patient and treated with methyl prednisolone pulse. \n\nConclusion: Kidney allocation for sensitized patients from deceased donor based on virtual cross match may help in achieving better improved graft function and prevent hyperacute rejection.",
    "Rajesh Sawant, Anand Bodade, Shantanu Sen, Sameer Tulpule \n\nKokilaben Dhiruben Ambani Hospital, Mumbai, India. \n\nIntroduction: While donor specific anti-HLA antibodies (DSA) have been implicated in graft rejection in solid organ transplantation, their role in hematopoietic stem cell transplantation (HSCT) remain unclear. \n\nAim: To study the role of DSA for proper donor selection and its effect in the setting of allogenic HSCT. \n\nMethodology: HLA A, B, C, DRβ1 and DQβ1 high-resolution typing and DSA cross-matches of patients (n=40) and their prospective donors (n=64) were assessed. A case-control study was designed retrospectively to evaluate effect of pre-existing DSAs on engraftment. Eighteen cases with 5/10 HLA alleles mismatches and twenty two controls were selected for comparison. These were matched for disease, graft type, conditioning regimen, age, gender, blood group and sensitizing events. DSAs were tested with solid-phase assay (Luminex 100/200 platform).\n\nResults: DSAs were detected in five of forty patients (12.5%) however donors selected for transplantation were all negative for DSA cross-match. These five patients who underwent haploidentical transplant had antibodies against Class II and one patient carried antibodies against both classes. A patient who experienced primary graft failure had second haploidentical transplant. No other known factors that could negatively influence engraftment were associated with the development of graft failure in this patient. \n\nConclusions: DSAs are associated with graft rejection in patients undergoing haploidentical stem cell transplantation. Anti-HLA sensitization should be evaluated routinely in hematopoietic stem cell transplantation with HLA mismatched donors for a better outcome.",
"Chandramouleeswaran N, Aruna Devi S, Nirmal Kumar M, Yogesh N, Srinivasan P, Saranya N, Vani S \n\nJeevan Stem Cell Foundation, India. \n\nIntroduction: Next-generation sequencing offers advantages for high resolution HLA genotyping with increased throughput and reduced genotype ambiguity. NGS typing can be performed using different platforms. In the present study HLA typing was standardized in Illumina Miniseq platform. \n\nMethods: 1210 genomic DNA samples were typed for HLA-A, -B, -C, -DRB1 and -DQB1 loci using MiniSeq™ and the data was analyzed by MIA FORA™ NGS FLEX software v3.1. \n\nResults: More than 90.53% of paired-end sequencing base calls exceeded Q30. Cluster density ranged between 140 to 300 K/mm2 with 87% clusters passing the filter. It was observed that as the cluster density increased the clusters passing the filter decreased and also that with increase in clusters passing the filter there was an increase in the valid barcodes that were read. 95.96% of alleles were assigned automatically by the software and 2.5 % of the alleles were assigned manually while 1.5% of the alleles remained unassigned. Percentage of automatic calls and the read counts significantly decreased when 96 samples were processed when compared to 48 samples (P=0.0037). We are in the process of identifying the root cause. \n\nConclusion: Based on the results obtained, certain standard parameters were set to analyze and assign the alleles. This study helped in understanding the advantages and limitations of the platform and software algorithm. \n\n",
"Prashant Pandey, Amit Pande \n\nTransfusion Medicine and Transplant Immunology, Jaypee Hospital, Noida-201304, India. \n\nIntroduction: during last one decade, as a result of improved desensitization therapy, there has been tremendous increase in the number of kidney transplantations in patients having anti-HLA antibodies (HLA sensitized) with excellent success rate. In present study, we are describing successful role of desensitization in 15 HLA sensitized patients having preformed donor-specific HLA antibody (DSA).  \n\nMethods: All patients were desensitized with pre-transplant single dose rituximab, therapeutic plasma exchange (TPE) and low dose IVIG. TPE was started using COM. TEC (Fresenius Kabi, Germany) after 21 days of administration of Rituximab. Complement dependent cytoxicity cross match (CDC-XM), luminex cross match with donor lysates (LM-XM), flowcytometry cross match (FC-XM) and panel reactive antibody (PRA) was done in all cases. If any of the four tests was positive single antigen bead assay (SAB) was performed. Desensitization therapy was given in 15 cases where DSA was detected. \n\nResults: Present study demonstrated the utility of including LM-XM and FC-XM in pretransplant compatibility testing algorithm as CDC-XM alone was not able to detect HLA antibody in 20% cases. The number of pre transplant TPE procedures required was 4.5(2-11). Post transplant six patients required TPE. Patients’ were discharged after seven (median) days of hospital stay. The longest duration of follow up was 2.5 years with mean duration of one year. Thus, at one year (mean) of follow up graft survival is 100%.  \n\nConclusion: Kidney transplantation in HLA sensitized patients using desensitization therapy is a useful tool to bridge the gap of demand and supply of organ.    \n\n","Ajay Kumar Baranwal, Deepali Bhat, Sanjeev Goswami, Gurvinder Kaur, Sanjay Agarwal, Narinder Mehra \n\nCommand Hospital, Pune, India. \n\nIntroduction: Development of antibodies in the recipient in the pre and post-transplant period is one of the major factors that influence the graft outcome. Their monitoring at regular intervals is an important pre-requisite. \n\nMethods: Pre and post transplant (postoperative day 7, 30, 90, 180 and the time of biopsy) sera from a total of 206 primary live donor renal transplant recipients were tested retrospectively for IgG anti-HLA antibodies using single antigen bead assay on a Luminex platform. All transplants had a mean follow up of 17.37±6.88 months.   \n\n                                                                                                                                                                             Results: A total of 55 (26.7%) recipients developed anti-HLA antibodies. Among the HLA antibody positive group, 18 (32.7%) were found to have donor specific antibodies (DSA) while the remaining 67.3% patients had no DSA. Sixteen patients developed AMR while ACR was observed in 13 cases. 75% of AMR cases had anti-HLA antibodies while only 15.4% of ACR cases and 23.2% of the no rejection group (NRE) were found to be positive for HLA antibodies. 100% of anti-HLA antibody positive AMR cases and 14.6% of antibody positive NRE patients had DSA while none of the ACR cases had DSA. HLA-DSA was highly significant in causing AMR (p less than.001).     \n\n                                                                                                                                                                     Conclusion: DSA negative anti-HLA antibodies is not harmful and regular monitoring for DSA both at pre-&amp; posttransplant stage is a necessity.    \n\n",
"Blessymol Varghese, Sam A, Tunny S, Vinoi D, Santhosh V, Dolly D \n\nCMC Vellore, India. \n\nIntroduction: The role of anti-MICA antibodies in renal allograft rejection is emerging and limited data is available from India. This study was done to look into the prevalence and clinical impact of anti-MICA antibody in a cohort of renal allograft recipients.   \n\nMethods: This was a retrospective case control study with 44 controls and 45 cases. Controls were CDC and Luminex DSA negative renal transplant recipients who had good graft outcome. Cases were CDC and Luminex DSA negative with poor graft outcome characterised by ACR, AMR and delayed graft function. Anti-MICA antibody was assayed in the pre transplant sera using LABScreen mixedTM from One Lambda.   \n\nResults: Out of total of 89 patients 63 were Indians. The prevalence of anti-MICA positivity in the overall group was 65.20 % and amongst Indian patients was 65.07%. Pretransplant anti-MICA positivity was associated with poor graft outcome as evidenced by anti- MICA positivity in cases (82.9%) vs. controls (61.5%) (p= 0.032).   \n\nConclusion: Anti-MICA antibody in our patient population awaiting kidney transplantation is of higher prevalence and the mechanism of anti-MICA antibody production has to be further explored. As pre transplant anti- MICA positivity is associated with poor renal allograft outcome, including anti-MICA screening in pretransplant work up can be considered for better prognostication and tailoring immunosuppression.\n\n",
"Anisha Navkudkar, Jyoti Rajak, Manisha Tambe, Sunil Rajadhyaksha, Meenakshi Singh, Priti Desai \n\nDepartment of Transfusion Medicine, Tata Memorial Hospital, Mumbai, India. \n\nIntroduction: Platelet refractoriness is the failure to achieve an acceptable increment in platelet count following platelet transfusion at least on two occasions. It is often multifactorial and is divided broadly into non-immune and immune causes. Immune platelet refractoriness primarily due to HLA alloimmunisation poses a major challenge in platelet transfusion support which can be dealt with HLA matched or crossmatched platelets. Another alternative is to give platelets matched at HLA- epitope level.  \n\nMethods: The eplet version of HLA Matchmaker represents a more complete collection of HLA epitopes and provides an elaborate assessment of HLA compatibility. We report here three patients who responded to HLA-epitope matched platelet transfusions from unrelated healthy donors.  \n\nResults: Duquesnoy antigen match grade for the three patients were B1X, D and D while Panel reactive antibody (PRA) was 23%, 8% and 85% respectively. Corrected count increment (CCI) within 10-60 minutes of unmatched platelet transfusion were 1600 and 2667 in first patient; 4800 and 3200 in second patient and 1200 and 3200 in third patient on two consecutive occasions. CCI within 10-60 minutes of epitope matched platelet transfusion were 12,750; 21,000 and 12,000 respectively. \n\nConclusion: HLA epitope matching approach in immune refractory patients can have very impressive 1 hour CCI results. It can be expected to benefit platelet transfusion outcome and increase the number of compatible donors for refractory patients.\n\n",
    "Shalaka Kadam, Selma Zenia D’Silva, Anand G Bodade, Sunil B Rajadhyaksha, Meenakshi SinghHLA and Immunogenetics Lab, Department of Transfusion Medicine, Tata Memorial Hospital, Mumbai, India.Introduction: The present case control study was initiated to examine the association of HLA-KIR ligands in myelogenous leukemias in Indian Population. \n\nMaterials and Method: The CML group included 253 patients and 1186 donors, whereas, the AML group included 453 patients and 2063 donors. HLA typing and KIR genotyping was performed by PCR-SSP. The genotype frequencies of KIR ligands for HLA-C and HLA B were compared in patients and controls. The disease and HLA-KIR association was studied by calculating odds ratio (OR), 95% confidence intervals (CI), χ2 test and values with p less than 0.05 were considered to be statistically significant.  \n\nResults: Significant positive associations were observed with HLA B, C and DQB1 in CML patients wherein HLA-B*55, HLA-C*01, *03, *04, HLA-DQB1*02 and*03 were higher as compared to controls, and HLA C and DQB1 in AML patients wherein, HLA-C*14, HLA-DQB1*05 and DQB1*06 alleles were higher as compared to controls. According to haplotype frequencies, HLA-A*01/B*15/DR*07, HLA-A*01/B*15/DR*15, HLA-A*02/B*40/DR*01, HLA-A*02/B*40/DR*15, HLA-A*68/B*15/DR*15 and HLA-A*68/B*52/DR*14 were significantly higher in CML patients than in controls. A significantly higher number of AML patients co-expressed the C2C2 epitope and missing epitopes for BW4 ligand T-I- as compared to controls. \n\nConclusions: The importance of this retrospective study is to define the role of HLA alleles and KIR ligands in predisposition to leukemia.    \n\n",
"",
"Vani S, Aruna Devi S, Chandramouleeswaran N, Yogesh N, Nirmal Kumar M, Mini N, Saranya N, Srinivasan P.\n\nJeevan Stem cell Foundation, Chennai.\n\nIntroduction: The Major Histocompatibility Complex (MHC) region has consistently emerged as the most polymorphic region in the human genome, with the advent of Next Generation Sequencing (NGS) technology unambiguous HLA typing at 4 field level has become possible.\n\nMethods: In this study, NGS was employed for typing HLA A, B, C, DRB1 and DQB1 in five different linguistic groups namely Tamil (n=7215), Kannada (n=174), Telugu (n=186), Urdu (n=397) and Malayalam (n=356) from South India. We compared the allele and haplotype frequencies at three field level among these five groups.\n\nResults: Most frequent and unique 5 locus extended haplotype analysis brought out that A*01:01:01~B*57:01:01~C*06:02:01~DRB1*07:01:01~DQB1*03:03:02 is the most frequent haplotype found in the South Indian linguistic groups (HF 2-4%). With an increase in the number of samples, the probability of finding population specific haplotype increased. It was observed that with increasing sample size all the haplotypes that had a frequency of >0.05%, were all in significant LD at their consecutive two locus haplotypes.\n\nConclusion: It becomes mandatory to have a multi-linguistic Indian national registry in order to find the more prevalent and population specific 5 locus haplotypes amongst the Indian population. This might help to find a better match within and among different populations.\n\n",
"Samrat Thapa, Ankit Mathur\n\nBangalore Medical Services Trust (BMST)\n\nBackground: Before kidney transplantation, recipients are routinely screened for preformed antibodies and prospective cross-matches. The methods used to detect donor-specific antibodies (DSA) are CDC cross-match, Flow cross-match, Luminex Lysate cross-match &amp; Single Antigen Bead Assay(SAB). In this study, we compared Luminex lysate crossmatch with SAB to detect HLA DSA.\n\nAim: The objective of this study is to analyze co-relation between Luminex lysate Cross-Match &amp; SAB Assay\n\nMethod: We studied all the samples received for SAB from January to December 2017. All the patients registered for renal transplant underwent CDC Cross match, HLA typing &amp; Luminex Lysate Cross-match. The patients whose Luminex cross-match found positive were further analyzed by SAB assay to detect DSA after discussing with Nephrologists. Anti-HLA antibodies were identified by SAB &amp; DSA was detected using donor’s HLA typing.\n\nResults: We received 343 Pre Transplant cases and total 20 positive Luminex cross-match cases underwent SAB. HLA DSA was found in 12(60%) cases but in remaining 9(40%) cases no DSA was detected. It was also observed that 37.5% of Class I positive discordant cases and 33.33% of Class II positive discordant cases had MFI value >3000.\n\nConclusion: Luminex Lysate cross-match is highly sensitive method to detect donor specific antibodies and cost effective also. However, specificity needs to be further evaluated. Single Antigen test is found best method to detect DSA &amp; it shows good clinical correlation too.\n\n","Siddharth Mittal, Sam Arul Doss, Stephen S, Dolly Daniel\n\nDepartment of Transfusion medicine &amp; Immunohaematology, Christian medical college, Vellore\n\nIntroduction: Isolated T-cell positive flow-cytometric crossmatches (FCXM) are generally considered as non-specific and clinically-insignificant. HLA-class I antibodies are expected to give a combined T- and B-cell positivity. However, upon repetitively observing this phenomenon in our laboratory, we decided to compare it to the Single antigen bead (SAB) assay on Luminex to ensure that there was no donor-specific antibody (DSA).\n\nMethods: This was a retrospective analysis of 19 patients with isolated T-cell positive FCXMs (X-median ratio of Test/Negative-control >1.0). Using stored anti-sera, SAB assay on the Luminex was performed. An MFI between 500 and 1500 is considered to be borderline positive and an MFI > 1500 as positive.\n\nResults: Of the 19 patients, a SAB assay could only be performed on 15. Among these 15 patients, 7 were negative for DSA on the SAB assay, while 6 were positive for DSA on the SAB assay with MFIs ranging between 544 and 8959. The other 2 showed CREG/Broad antigen reactivity towards the mismatched donor antigens.\n\nConclusion: The dogma that isolated T-cell positive FCXM is non-specific in etiology needs to be revisited. Identification of DSA or CREG/broad antigen reactivity in more than 50% of the patients exhibiting this phenomenon, raises the issue of having to pursue the search for DSA in these patients as well. While the reason for this remains unclear and could be multi-factorial, considering the clinical relevance of DSA, it is critical that their presence or absence be defined by SAB assay to ensure best quality of patient-care.\n\n",
"Stephen S, Siddharth Mittal, Sam Arul Doss, Dolly D\n\nDepartment of Transfusion Medicine and Immunohaematology, Christian Medical College Vellore.\n\nIntroduction: Autoantibodies to HLA antigens usually belong to IgM isotype and are considered non-detrimental to graft survival. However, it would be unwise to assume them as innocuous always as there are reports of them being IgA/IgG in nature and/or clinically significant. Here, we describe a case report of IgG HLA auto-antibody in a 46 years old female with Myelodysplastic syndrome being evaluated for stem cell transplant with brother as the prospective donor.\n\nMethods: Patient and donor HLA typing was performed by NGS. Crossmatch testing was performed by flow cytometry using DTT-treated recipient’s serum and donor lymphocytes. Antibody specificity testing was conducted by Luminex Single antigen bead assay (SAB).\n\nResults: Donor recipient flow cross-match (FCXM) was both T- and B-cell positive. Auto FCXM was also performed which revealed negative T- and positive B-cell FCXM. SAB was positive for Class I but no DSA was present. Class II SAB was also positive with presence of auto-antibodies against HLA-DRB1*14:04 (MFI:2792), DRB1*04:03 (MFI:20,117), DQB1*03:02 (MFI:5265). There was sharing of HLA-DRB1*04:03 and DQB1*03:02 antigens with the brother. There were no antibodies against the mismatched HLA antigens in the donor.\n\nConclusion: This case report demonstrates the present of auto-antibody in the recipient using two platforms: Flowcytometry and SAB. The presence of high-titer autoantibodies cross-reacting with donor lymphocytes along with strong sensitizing history of the recipient, prompted us to look for alternate donors preferably unrelated.\n\n",
"Sam Arul, Doss. R, Siddharth Mittal, Dolly Daniel\n\nDepartment of Transfusion Medicine &amp; Immunohematology Christian Medical College, Vellore.\n\nIntroduction: Rituximab (RTX- Anti-CD20), a frequently used monoclonal antibody in transplants across the ABO barrier/highly sensitized interferes with standard CDC and B-cell flow-cytometric crossmatch (FCXM). However, we have observed positive T-cell FCXM in this setting. We aimed to study the effect of RTX on T-cell FCXM on non-pronased (NP) and pronased (P) donor lymphocytes and compare it to single antigen bead (SAB) assay results to assess donor specificity.\n\nMethods: This is a retrospective single centre study in which FCXM and SAB results were analyzed for 16 renal and 2 stem cell transplant (TX) recipients treated with RTX.\n\nResults: A total of 18 RTX treated TX recipients with 21 FCXM and SAB were evaluated. Using NP donor lymphocytes, all 21 FCXM’s were strong B cell positive which significantly reduced with P donor lymphocytes cell positivity was observed in 9/21 FCXM in the NP group with a typical T cell tailoring phenomenon. This did not persist in the P treated group.SAB assays were negative for both HLA class I and II antibodies in all these patients.\n\nConclusion: Though traditionally RTX is described to impact on B cell FCXM we describe a consistent finding of T cell positivity with tailoring phenomenon in 43% of patients tested, which clears with pronase treatment. The Negative SAB in all these patients reinforces the fact that this is a false positive finding and should not be misconstrued as a possible class I donor specific antibody. Structural homology of Fc receptors on T cell to CD20 could be a possible explanation of the same.\n\n",
"Bhavsar S, Trivedi. V.B., Dave A.P., Patel J.R., Pandya K.G, Bhindora M.J, Thakar K.N , Modi .P.D\n\nTransplantation immunology Laboratory, G. R. Doshi and K. M. Mehta Institute of Kidney Diseases &amp; Research Centre (IKDRC)- Dr. H.L. Trivedi Institute Of Transplantation Sciences (ITS), Civil Hospital Campus, Asarwa, Ahmedabad\n\nABSTRACT: Influence of de novo HLA DQ antibodies on graft outcome. We evaluated644 kidney transplant recipients for donor specific antibodies during 2013-14.Renal transplant workup has evolved HLA typing, complement dependent cytotoxicity (CDC), Flowcytometry cross match , solid phase immunoassay and single antigen bead assay (SAB). Post-transplant de-novo antibodies have been monitored using single antigen bead assay and serum creatinine level. Fluorescence intensity of these beads determined by using Luminex platform. In the total patients, post KT DSA was found in 104 patients (16.14%).Out of these 104 recipients class I and class II donor specific antibodies were found in 53 patients (8.21%), DSA class I found in 28 patients(4.34%), DSA class II found in 76 patients (11.80%).Among them DQ-dnDSA was most prevalent in 72 patients (11.16%). With 4 year follow up, developing DQ-dnDSA was associated with rise in creatinine in 34 patients (5.27%) with Mean Fluorescence Intensity between 1136-15,529. Therefore monitoring of DSA de-novo HLA DQ may help in early detection of allograft rejection and in improving long term allograft outcome. Despite the advances in transplantation, HLA antibodies are major hurdle. These cases favor the need for extended HLA typing (DQ) to closely monitor. Post-transplant detection of DQ-dnDSA antibodies are associated with poor graft outcome which are strongly predictive transplant failure.\n\n",
"Yoginder Pal Singh1, Arup Ratan Dutta2, Narinder K Mehra1,3\n\n1Centre for Excellence in Transplant Immunology, SRL Laboratories Gurgaon, Haryana, 2Fortis Hospital Anandapur, Kolkata, 3All India Institute of Medical Sciences, New Delhi\n\nEvaluation of HLA antibodies with solid phase immunoassays has assisted in defining donor specific antibodies (DSA) relevant for patients with end stage disease requiring an organ transplant. However, realization of these benefits requires careful and accurate interpretation of the high end sensitive test results. It is also important to differentiate an inert from deleterious antibodies.\n\nWe report here two cases representing two different ethnic groups referred to our center for pre-transplant workup for DSAs. First case was a highly sensitized recipient with an HLA- haploidentical related living donor form African ethnicity. Second case was a moderately sensitized recipient with unrelated living donor (one allele match at HLA-B &amp; DR loci) from Asian ethnicity. In both cases pre-transplant serum of the recipients was tested by three assays: (i) CDC-XM (ii) FC-XM (iii) Single Antigen Bead. The CDC-XM and FC-XM tests appeared negative in both cases, while the SAB test results were positive with very high MFI values. All three assays were repeated in both cases after ten weeks and similar trend was observed with almost similar elevated MFI values with SAB assay.  We tried to resolve all possibilities of interferences in sera which could lead to false positive or false negative results. The sera were pretreated with various methods like heat inactivation, EDTA and commercially available Seraclean. When tested again for SAB assay, although a general drop of 15-20% in MFI values was observed, the MFI values were still in the range of 15-17000. We performed C3d test using Immucor kit in both cases to decipher the complement binding status of HLA antibodies. The results for first case revealed a total lack of complement binding antibodies and the patient is still awaiting for the transplant. Whereas, the second case showed very high complement binding DSAs against HLA-A*11:01, an allele mismatched between the donor and recipient. The patient has been successfully transplanted and is under constant monitoring. This raises the question on clinical relevance of various assays to differentiate between the inert versus deleterious antibodies. Our experience also raises the question whether all antibodies are clinically relevant. Thus HLA antibody assessment requires careful evaluation of solid phase assay results in the context of complement testing and the patient\'s sensitization history. New solid phase assays like C3d could provide useful information on antibody status and advance our understanding on allograft rejection.\n\n",
"Mehta P1, Naker D1, Sawant R1, Sheth S2\n\n1 Department of Histocompatibility and Immunogenetics,  2 Department of Nephrology, Kokilaben Dhirubhai Ambani Hospital, Mumbai\n\nBackground: A false positive CDC XM has a negative impact in donor selection process obliterating healthy, donor compatible population.\n\nCase report: 47Y, male with chronic kidney disease was planned for a ABO compatible renal transplantation from his sister. CDC and DSA cross match were negative 10 days prior to transplant. The 48 hrs pre transplant CDC XM showed 40% positivity. Plasma exchange and IVIG were used for desensitization and resulted in 15% positivity in CDC XM. DSA Class I and II and HLA antibody screen were negative. Patient’s indirect anti globulin test was positive and anti-M antibody (Ig G + Ig M) was identified. Rituximab was added to desensitization regime and anti- M titre reduced to 8 from initial titre of 256.The transplant was done and graft function is excellent till date (10 months post transplant).\n\nConclusion: Red cell allo-antibody should be considered in differential diagnosis of a positive CDC cross match. The utility of DSA as a pre transplant monitoring tool is immense in such situations. Institutional policies regarding plan of action in the event of positive CDC XM and negative DSA XM and vice versa should be formed.\n\n",
"Naker D, Sawant R\n\nKokilaben Dhirubhai Ambani Hospital, Mumbai\n\nBackground: The detection of antibodies prior to transplantation is an important step in assessment of patient immunological risk and exclusion of incompatible donors. Many centres have now implemented DSA along with CDC cross match for renal transplant cases.\n\nCase Report: 34Y, male with end stage kidney disease was referred for an ABO compatible transplant from mother. The CDC crossmatch done 30 days prior to transplant was negative. DSA crossmatch was negative for class I (MFI 189) and positive for class II (MFI 1671). Since CDC and DSA class I were negative, the nephrologists went ahead with the transplantation. On day 6 post transplant, serum creatinine showed a rising trend (upto 2.13mg/dl) and therefore renal biopsy was done which showed mild acute tubular necrosis with positive C4d staining. DSA XM performed on day 15 post transplant showed negative class I (MFI 148) and positive class II (MFI 9987) confirming AMR. Patient was started on steroids and IVIG and serial plasma exchanges were performed. Then DSA class II levels came down to 1602. DSA levels have been monitored periodically and class II MFI values have been ranging from 2000-4000. Patient is maintained on routine immunosuppression and graft is intact with serum creatinine level between 1.7-1.8mg/dl 8 months post transplant.\n\nConclusion: DSA isolated class II positivity in renal transplant recipients correlates strongly with AMR and should be considered clinically significant\n\n",
"Divyank Saini1, Dong-Feng Chen2, Wendy E. Hanshew2, Barbara Ure2, Collin Brack2\n\n1HLA Data Systems, 2Duke University Medical Center - North Carolina USA\n\nOur aim is to evaluate the utility of incorporating epitype analysis into the computerized virtual crossmatch (vXM) workflow for donors with DP alleles that are not covered by a recipient’s HLA antibody screening and identification assay. Our lab has used vXM to select deceased donors for transplantations for more than ten years. If the donor DP antigens are covered by the assay, the DSA can be determined directly by vXM. If the donor antigen is not included in the assay but is covered by an epitype (EDP) group which includes a DP antigen covered by the assay, the DSA can be determined based on the positivity of the EDP antigen using manual epitype profiling. A new epitype profiling system was deployed by informatics vendor HLA Data Systems, developers of the mTilda Lab Management System and VxMatch, to ensure that DP alleles not covered by HLA antibody tests are incorporated into an electronic vXM workflow.\n\n",
"Esther Philip, Sanjeev Goswami\n\nDept. of T.I.I, All India Institute of Medical Sciences, New Delhi – 110029\n\nIntroduction: The role of donor specific anti-HLA antibodies (DSA) in graft failure and rejection has been a subject of investigation world over. We have analyzed the post-transplant antibody profile in serum of renal transplant recipients who had AMR.\n\nMethods: Kidney recipients with acute rejection episode and biopsy indicating C4d+ were selected for the study (n=32). Those with well-functioning grafts served as the control group (n=27). Serum from these recipients was analyzed using Single Antigen Bead kit (One Lambda/Immucor) for the presence of anti-HLA (class I/II) antibodies.\n\nResults: 30 patients undergoing graft dysfunction showed the presence of anti-HLA anti- antibodies with 7 samples being positive for HLA class I and 22 samples were positive for HLA class I and II. Only three samples were positive for HLA class II alone. Presence of anti-HLA class II antibodies appears to be associated with an greater risk of antibody mediated rejection (AMR), (25/32 cases, p &lt; 0.05).\n\nConclusion: Antibody mediated rejection is an irreversible process and allograft recipients who had AMR appear to develop more of anti-HLA class II antibodies\n\n",
"Mahendra Narain Mishra\n\nHLA antibody mediated rejections are less common, but more severe than cell mediated rejection. Numerous technical issues are associated with their detection and specification. Issues Luminex based assays may be associated with high background, high reactivity against self-alleles, failure to detect reactivity against alleles from previous transplant, naturally occurring antibodies and false positive reactions. Difficulties in use of Flow crossmatch include difference cut offs and methods of reporting. It may be reported as channel shift, relative median fluorescence or simply as positive / negative. This non -uniformity in reporting may lead to some recipients being denied a transplant and rarely to unexpected rejections. Solutions Heat treatment, addition of EDTA, addition of foetal calf serum, meticulous validation of kits, additional wash and use of adequate numbers and range of internal quality controls and replicate testing may be used. Many centres abroad use products from both available vendors for confirmation of antibodies. Cost cutting is possible by validating with half volume reagents.\n\n",
"Suchita Jogale, Rajeshwari Basavanna, Anand Deshpande\n\nP D Hinduja National Hospital &amp; M R C, Mumbai\n\nAnti idiotypic antibodies to cytotoxic HLA antibodies are known to favorably influence renal transplant outcome, however the presence of cytotoxic anti HLA antibodies against donor antigen is detrimental to the graft. Here we describe a case, which highlights the importance of identifying the presence of anti idiotypic antibodies, so that transplant can be carried out in spite of positive CDC.\n\nCase report:\n27 years old lady with CKD due to HTN during pregnancy referred for pre-transplant work-up with her aunt as a prospective donor. No history of sensitization except one pregnancy.  CDC was positive (80%), and luminex cross match (LXM) was positive for class I and negative for class II. T &amp; B cells CDC with DTT treated serum showed the presence of Allo - IgG + IgM antibodies. Pre-transplant work up with mother (2nd donor), routine and auto CDC and LXM were negative, however, CDC with DTT treated serum was positive with B cells, raising the possibility of anti- idiotypic antibodies for class II. SAB for class I positive for multi-specific anti-HLA –B* antibodies and in class II (DTT treated) positive for DQB1*05:01 &amp; DQB1*06:01antibodies. Pre-Transplant work up with cadaveric donor, routine CDC and LXM was negative and positive with DTT treated serum.Patient ultimately underwent renal  transplant with cadaveric donor and doing well (one month). Further HLA typing of deceased donor was carried out, which showed presence HLA-DQB1*05:01 which is donor specific.\n\nConclusion – Patients with anti idiotypic antibodies, in absence of free IgG cytotoxic antibodies are candidates for transplant and have good transplant outcome.  This case highlights the importance of identifying these antibodies and their importance.\n\n",
"Rajeshwari Basavanna, Suchita Jogale, Anand Deshpande\n\nP D Hinduja National Hospital &amp; M R C, Mumbai\n\nIntroduction: Positive CDC is absolute contraindication for transplants.Clinical relevance of detecting allo/auto IgM alone during pretransplant work up is not clear. Studies report protective effect of IgM however, the other reports harmful effect on the graft.  With this background, we reviewed allo-IgM positive cases. We are also reporting one case, which highlights the importance of having consensus on  allo IgM antibodies.\n\nCase Report\n\n44 years old gentleman referred for pre transplant work-up withwife being a donor. He had   history of Polycystic kidney disease. There was no sensitization history.  CDC weak positive with 20% cell lysis, however luminex cross match was negative. In view of this, possibility of auto/ allo IgM antibodies was considered. Test repeated after 4 weeks showed CDC with strong positivity (80%), Auto CDC, LXM and PRA screening were negative. CDC with DTT treated serum was negative confirming the presence of allo-IgM antibodies with no isotype switch.  When patient was cross-matched with brother, CDC was weakly positive (15%) and CDC with DTT treated serum was negative. SAB for Class I was negative and Class II was positive for DRB4*01:01 antibody which is non-donor specific. Patient was worked up for swap and cadaveric transplants and 5 cycles of plasmapheresis was done to remove IgM antibodies, meanwhile patient developed infection and succumbed. We also had seven other patients with allo-IgM in the last 16 months awaiting transplant.\n\nConclusion- Generally IgM are considered innocuous and IgG are detrimental. Multiple small number of studies done shows allo-IgM antibodies are not detrimental to the graft. We need to have general consensus with nephrologists to manage patients with all-IgM which are specific or/and non-specific to donor HLA antigens.\n\n",
"Giriprasad Venugopal1, Balakrishnan S Ramakrishna1, Alka Singh2, Govind K Makharia2\n\n1SRM Institutes for Medical Science, Chennai,  2All India Institute for Medical Sciences, New Delhi\n\nIntroduction: Celiac disease (CeD) is common in several parts of the world, including India, the disease affects approximately 1% of the north Indian population. CeD is reported to occur only in genetically predisposed individuals who express the human leukocyte antigens (HLA) DQ2 and/or DQ8. We evaluated the HLA DQA1-DQB1 genotypes and haplotypes frequencies in patients with CeD from Indian population.\n\nMethods: Peripheral venous blood was obtained from 400 patients with CeD. DNA was isolated and high resolution typing of HLA-DQA1 and HLA-DQB1 loci was performed. Amplifying these genes using long range PCR and multiplex sequencing on the Illumina platform. The assembled sequences were matches against the IMGT/HLA database and HLA DQ genotypes and haplotypes were assigned on basis of best match.\n\nResults: The DQ2.5 (DQB1*0201-DQA1*0501), DQ2.2 (DQB1*0202-DQA1*0201) and DQ8 (DQB1*0302-DQA1*0301 or DQB1*0302-DQA1*0302) haplotypes were found in 36.75%, 12.75% and 8% of CeD patients respectively. Thirty one patients (7.75%) with CeD were homozygous for DQ2.5 and 6 (1.5%) CeD patients were homozygous for DQ8 haplotypes. The frequencies of DQ2.5/2.2, DQ2.5/DQ8 and DQ2.2/DQ8 haplotypes were 6.25%, 4.5% and 1.2%, respectively. Overall, 297 (74.25%) CeD patients had genotypes which were capable of expressing DQ2 and/or DQ8 antigens.\n\nConclusion: The genotypes capable of expressing DQ2 and/or DQ8 antigens were found in only 74.25% in Indian CeD patients.\n\n",
"Gaurav Sharma1, Veron Ramsuran2, Gurvinder Kaur1, Narinder Mehra1\n\n1All India Institute of Medical Sciences, New Delhi, 2Centre for Cancer Research, National Cancer Institute, Frederick, MD, USA\n\nIntroduction: Major Histocompatibility Complex (MHC) is the most polymorphic gene dense region of the human genome. Indeed; this region represents a mini genome model with strongest genome wide association to several autoimmune and infectious diseases as well as cancers; however the mechanisms remain largely unknown. Besides variable and promiscuous peptide binding properties of various Human Leukocyte Antigens (HLA) alleles; unexplored mechanisms of allele specific differential HLA expression need to be investigated. The same can partly be controlled and defined by variants located within the promoter regions of the HLA genes.\n\nMethods: We analyzed ~2 KB promoter region of a total of 68 alleles from 57 major lineages of classical HLA class I genes (A, B and C) by direct sequencing. Comprehensive genomic and phylogenetic analysis was performed and the influence of promoter variability on allele specific HLA expression was evaluated.\n\nResults: The overall nucleotide diversity within this promoter segment roughly follows that seen within the coding regions (HLA-B>HLA-A>HLA-C). Total 57 novel sequences representing major HLA-A, B and C lineages were submitted to genebank database with accession no. KY427751-KY427809. These sequences serve as reference for exploring potential HLA expression mechanisms.\n\nConclusion: Promoter sequence divergence might impact promoter activity partially and thus differential expression between classical HLA class I lineages to certain level.\n\n",
"Ajeet Alwadhi, Mohd. Abid, Shashi Kapoor Gautam, Akanksha Sharma, Gaurav Sharma, Uma Kanga,\n\nDepartment of Transplant Immunology and Immunogenetics, All India Institute of Medical Sciences, New Delhi-110029, India\n\nIntroduction: Hypersensitivity to anti-retroviral drug abacavir has been consistently reported to be associated with HLA B*57:01 in various populations. As a result, screening for this risk allele has been made mandatory across various regions worldwide. Despite advances in therapeutic and preventive care, HIV infection continues to be prevalent and availability of diagnostic facility for detecting HLA alleles associated with adverse drug reactions could be beneficial for management of patients. Present investigation was undertaken, as the relevance of the HLA-B*57 screening towards abacavir hypersensitivity is largely unknown in North Indian population.\n\nMaterial and Methods: HLA typing records of a total of 883 healthy individuals were screened for the presence/absence of HLA-B*57 allele. All of these samples were genotyped for HLA-A and B by rSSOP (One lambda/ Immucor) method.\n\nResults: Our analysis revealed a high occurrence of B*57 in North Indians. About 11% individuals were observed to be positive for this allele in North Indians. Further B*57:01 frequency in this cohort is being investigated.\n\nConclusion: The observed preponderance of HLA-B*57 in North Indians clearly suggest the possible risk for abacavir hypersensitivity and hence the relevance of screening for B*57:01 allele before prescribing this drug to HIV infected individuals. Additionally, there is also a need to establish a cost effective diagnostic assay for HLA-B*57:01 detection.\n\n",
"Snehil Kumar, Sam Aruldoss, Nitty S. Mathews, Debashish Danda, John Mathews, Dolly Daniel\n\nChristian Medical College, Vellore\n\nIntroduction: HLA-B*27 is a class I molecule of the MHC which is strongly associated with Ankylosing spondylitis (AS). There is limited literature regarding the prevalence of HLA-B*27 subtypes in the ethnic Tamil population. This knowledge can help to analyze the distribution of HLA-B*27 subtypes and correlation between HLA-B*27 subtypes and the clinical features of AS.\n\nMaterials and Methods: Tamil ethnic adult patients with HLA-B*27 positivity were considered for the study. Diagnosis of AS was based on ASAS criteria. Clinical parameters were retrieved from the patient information system. Controls were healthy adult Tamil donors for solid organ or stem cell/bone marrow transplant.\n\nResults: This study included 113 patients with AS and 100 healthy controls. The predominant subtypes were HLA-B*27:05 and HLA-B*27:04. All the subtypes showed disease predilection for males. Statistically significant association of mixed involvement was seen with subtype HLA-B*27:04(pvalue: 0.0295). Knee and ankle joint involvement in females (pvalue: 0.0425 and 0.0049) with HLA-B*27:04subtype was statistically significant. HLA-B*27:05 positive patients demonstrated a significant association(pvalue:0.0207) with uveitis.\n\nConclusion: Our study has established, for the first time, the prevalence of HLA-B*27 subtypes in Tamil population with AS. Our data suggests a varied association of various subtypes with clinical features of AS from other ethnic groups. This data may prove helpful in early diagnosis and early intervention for ankylosing spondylitis in Tamil ethnic population.\n\n",
"Sanghamitra Machhua1, Yashwant Kumar1, Shefali Khanna Sharma2, Heera Singh1, Surjit Singh3, Ranjana W Minz\n\nDepartment of Immunopathology1, Department of Internal Medicine2, Department of Pediatrics3, Post Graduate Institute of Medical Education and Research, Chandigarh, India.\n\nBackground: Systemic sclerosis (SSc) is an autoimmune connective tissue disease of unknown etiology whose pathogenesis is characterized by three hallmarks: vascular damage, autoantibodies production, and fibroblast dysfunction causing excessive collagen deposition in skin and internal organs, clinically seen as fibrosis. There are two subsets of SSc depending upon the degree of skin involvement, autoantibody association and pattern of organ involvement, limited cutaneous systemic sclerosis (lcSSc) and diffuse cutaneous systemic sclerosis (dcSSc).Various studies across the world reported promising genetic association with SSc, however no such genetic study has been reported from India .This study was planned to explore the HLA association with SSc in north Indian population.\n\nMethods: 40 (22-lcSSc and 18-dcSSc) patients and 99 healthy volunteer controls were enrolled in the study from same ethnicity. ANA were detected in sera of patients by indirect Immunofluorescence. HLA typing was done for DRB1 and DQB1 loci by PCR-SSP method.\n\nResults: DRB1*15(P value-0.013, odds ratio-2.217) and DQB1*03(P value-0.0023, odds ratio-2.967) were found to be significantly associated with SSc. There was no difference between 2 subsets of SSc in terms of HLA association.\n\nConclusion: Present study indicates that DRB1*15 and DQB1*03 are significantly associated with SSc patients of north Indian population, also earlier reported in Thai and European ancestry. For further confirmation large number of patient samples will be analyzed in near future.\n\n",
"Nitty S. Mathews, Sam Arul Doss, Dr. Dolly Daniel\n\nDept. of TMIH, CMC Vellore\n\nIntroduction: It is well known that anti-HLA antibodies can severely damage kidney allografts. Lately, antibodies against non-HLA antigens have been implicated in acute allograft rejection. We aimed to ascertain the implications of anti-angiotensin 1 (anti-AT1) receptor antibodies in cases of antibody-mediated rejection (AMR) and acute cellular rejection (ACR).\n\nMethods: Seventeen patients who were negative for anti-HLA antibodies by complement-mediated lymphocytotoxicity &amp; flow cytometric crossmatch pre-transplant, and had acute graft rejection between May and November 2017 were studied. Anti-AT1 receptor antibody testing was performed on the pre-transplant serum of 14 cases and post-transplant serum of 5 cases. Pre-transplant serum samples of 6 patients who underwent renal transplant during the same period and had normal graft function were also tested for anti-AT1 receptor antibodies.\n\nResults: Among the 17 cases of acute allograft rejection, 8 had AMR, 8 had ACR and 1 had AMR+ACR. All tested 14 cases and 6 controls had absence of anti-AT1 receptor antibodies in their pre-transplant serum samples. Five post-transplant serum samples from cases also tested negative for these antibodies.\n\nConclusion: Though reported to be a cause of acute graft failure in renal allografts in those with no anti-HLA antibodies, we could not demonstrate any case with anti-AT1 receptor antibodies in our setting. In these cases, other causes of non-HLA graft rejection such as anti-MICA and anti-endothelial antibodies need to be considered.\n\n",
"Rajni Rani1.  Archana Singh1, Samreena Mehtab1,2, Utpreksha Vaish1, H.K. Kar3, Ravinder Goswami2,3\n\n1National Institute of Immunology, ArunaAsaf Ali Marg, New Delhi.\n\n2All India Institute of Medical Sciences, New Delhi\n\n3Dr. Ram Manohar Lohia Hospital, New Delhi.\n\nIntroduction: Idiopathic Hypoparathyroidism (IH) and Vitiligo are autoimmune disorders where parathyroid gland and pigment producing melanocytes respectively are the targets of the autoimmune responses. We have earlier shown association of Major Histocompatibility Complex (MHC) alleles with the two autoimmune disorders. In order to understand the functional implications of these associations, we studied the MHC restricted T cells responses in peripheral blood mononuclear cells (PBMCs) of patients against self-antigens.\n\nMethods: MHC class-I and class-II alleles were studied in subjects with hypoparathyroidism, vitiligo and healthy controls using Luminex based PCR-SSOP assay.  MHC restricted peptides were evaluated for their ex-vivo potential to stimulate PBMCs in IFN-γ ELISPOT assays.\n\nResults:In IH HLA-A*26:01 and in vitiligo HLA-DRB1*07:01 were found to be the primary HLA alleles associated with the disorders. 82.2% of IH patients showed MHC class-I restricted CaSR specific responses in IH suggesting the presence of memory CD8+ T cells in the peripheral circulation of patients with IH. Similarly, 79.3% of vitiligo patients showed MHC class-II restricted melanocyte specific memory CD4+ T cells in peripheral circulation of vitiligo patients.\n\nConclusions:We demonstrate the affinity of different auto-antigenic peptides for predisposing MHC class-II and class-I molecules, which may be involved in orchestrating (through CD4+ T cells) and implementing the autoimmune responses (through CD8+ T cells) in vitiligo and hypoparathyroidism.\n\n",
"Shiva Krishna Katkam\n\nNizam\'s Institute of Medical Sciences, Punjagutta, Hyderabad, India\n\nIntroduction: Systemic lupus erythematosus (SLE) is a multifactorial, complex autoimmune disease characterized by the presence of numerous autoantibodies against nuclear components. The major histocompatibility complex (MHC) class II alleles have major contribution in susceptibility for SLE. MHC class II alleles HLA-DR3, DR1, DR4, and DR7 are associated with higher TNF-α synthesis where as HLA-DR2 and DR5 were found to be associated with lower TNF-α levels, presumably TNF-α promoter SNPs were in tight linkage disequilibrium with these alleles.\n\nMethods: Genotyping of HLA-DRB1 was performed by a polymerase chain reaction technique using the sequence-specific primer method. The TNF-α gene promoter region (from -250 to -1000 base pairs) was analyzed by direct Sanger’s DNA sequencing method.\n\nResults: HLA DRB1*07 was significantly associated with SLE (OR: 2.02; 95% CI 1.34-3.04, P=1.50×10-4, Pc =1.95×10-3), whereas DRB1*14 allele is negatively associated with SLE (OR: 0.49; 95% CI: 0.31-0.76, P =1.70×10-2, Pc=0.221). In addition, the HLA-DRB1*07/15 genotype tended to be positively associated with SLE (OR: 3.23, 95% CI: 1.57-6.63, P =0.0009).The -863A (rs179924) allele showed association with Lupus nephritis phenotype in patients with SLE (OR: 1.62, 95% CI: 1.04–2.53, P=0.034). The haplotype analysis revealed a significant protective association between TNF -863C, -857C, -806C, -646G, -572A and -308G (CCCGAG)  haplotype with LN phenotype (OR: 0.61, 95% CI: 0.41–0.92, P=0.02).\n\nConclusions: The present study is the first attempt to investigate the association of HLA-DRB1and TNF-α polymorphism with SLE. Both DRB1 alleles and TNF- α polymorphisms are found to influence the clinical manifestations of SLE and contributing to clinical heterogeneity and disease severity.\n\n",
"Manisha Agrawal, Sanjay Prakash\n\nR.N.T Medical College, Udaipur\n\nBackground and Objective Thalassemia is the leading haemoglobinopathy after sickle cell anaemia that accounts for 1.7% of global population. It is characterized by chronic anaemia requires monthly blood transfusion and regular iron chelation. Our aim was to determine awareness among parents of children with thalassemia major regarding the disease.\n\nMethods: One hundred twenty parents with thalassemic child on a regular multi-transfusion programmes were interviewed using a pre designed questionnaire. Questions regarding inheritance of disease, awareness regarding screening of thalassemia &amp; risk of repeated blood transfusion were asked.\n\nResults A total of 120 care takers questioned. Majority was of low socioeconomic class and 61.66% were illiterate with only 15% with higher education. Consanguinity was positive in 73.33% of the parents with extended family history of thalassemia positive in 39.16%.Knowledge regarding oral iron chelator therapy was present in 42.5% of the patient parent but only 15% were receiving it adequately. Only 45% were immunized against hepatitis B and 46.66% did not know whether they should be immunized. Only 12.5% knew that thalassemia is an inherited disorder and family screening of siblings and antenatal diagnosis in subsequent pregnancies was done in 8.3% and 7.5% respectively.\n\nConclusion: Thalassemia is a fairly common problem worldwide yet patients, parents and care givers are ignorant about it. Awareness of parents was inadequate. Education programmes and compulsory antenatal screening appear to be the order of the day.\n\n",
"Parmatma Prasad Tripathi, Ashish Jain, Neelam Marwaha\n\nDepartment of Transfusion Medicine Post Graduate Institute of Medical Education and Research, Chandigarh\n\nIntroduction: In ABO-incompatible allogeneic hematopoietic stem cell transplant (HSCT) monitoring for the appearance of donor derived ABO antigens on red cells (RBCs) is a valuable tool for deciding the transfusion strategy.\n\nMethods: It is a retrospective study (six months) where the patients undergoing ABO-incompatible (major/minor/bi-directional) allogenic HSCT were followed up for blood grouping, direct antiglobulin test (DAT) and antibody screen (ABS). Blood grouping was done by tube technique, DAT and ABS were done using gel technique (Bio-Rad, Switzerland). ABO-antibody titers were performed using tube technique.\n\nResults: A total of 6 patients underwent ABO-incompatible transplant including 3 major (1 case AB→B; 2 cases B→O), 2 minor (A→AB; O→B) and 1 bi-directional (A→B). ABS and DAT were negative. In major ABO-incompatible transplants, the recipient anti-A/anti-B titers varied from 2-16 (IgM:2-8; IgG:2-16) and in minor ABO-incompatible transplants the donor anti-B titer ranged from 8-32 (IgM:8-16; IgG:8-32). In the bi-directional case, the recipient anti-A titer was 32 (IgM=IgG). No plasma/red cell reduction was done in the product before the transplant. In one major ABO-incompatible transplant, a mixed-field (mf) reaction was observed with anti-A on day-44, with anti-B on day-31 in one minor ABO-incompatible transplant and with anti-A on day-36 in bidirectional ABO-incompatible transplant, which indicated red cell engraftment. A total of 30 (median:4.5) red blood cell units were transfused in the post-transplant phase.\n\nConclusion: Pre-transplant immunohematology work-up and the monitoring of blood group for mixed-field reactions are important tools for optimizing the transfusion strategy of patients undergoing ABO-incompatible allogeneic HSCT.\n\n",
"Patel Jayshree R, Trivedi V.B\n\nTransplantation Immunology Laboratory, G. R. Doshi and K. M. Mehta Institute Of Kidney Diseases &amp; Research Centre (IKDRC)- Dr. H.L. Trivedi Institute Of Transplantation Sciences (ITS),\n\nCivil Hospital Campus, Asarwa, Ahmedabad- 380\n\nThe presence of antibodies in kidney transplant is associated with an increased frequency of rejection and graft loss. This was two year prospective study to evaluate T and B cell cross match using flow cytometry (FCM) in living donor renal transplantation. Years of 2013 and 2014 patients 640 were evaluated using FCXM The flow cytometer measures cell-bound antibody with a fluorescing label, the amount of antibody being quantified by fluorescence intensity. The unit of intensity is called a channel, and the difference between controls and experimental is called mean channel shift. Although each laboratory sets its criteria for a positive test, a 50 mean channel shift for T cells and a 100 mean channel shift for B cells are our institute considered a positive test. Graft survival according to crossmatch results was compared by survival analysis Our studied included Six hundred forty patient out of which found to be FCM negative for T and B cells were 588 (91.8%). and 52(8.12%) of them were FCM positive, in which 27 (51.92%) were T + B positive, 6(11.53%) were T cell positive and 19(36.53%) were B cell positive. positive FXCM experienced significant reductions in 5-year graft survival\n\n",
"Mohd. Abid, Ajeet Alwadhi, Shashi Kapoor Gautam, Akanksha Sharma, Sunil Kumar, Uma Kanga\n\nDepartment of Transplant Immunology and Immunogenetics, All India Institute of Medical Sciences, New Delhi\n\nCase I- A 5 year old male suffering from aplastic anaemia was referred from Haematology department for HLA genotyping. We observed sharing of 4 out of 4 HLA-A, B alleles with one of his siblings and advised for HLA class-II (DRB1) testing. A mismatch was observed on DRB1 region, the patient and his sibling then appeared to be 5 out 6 match. Parental genotyping, at low resolution indicated that the father was homozygous for class I alleles. On performing high resolution HLA typing of the parents, allelic differences were observed in the two haplotypes of the father, which meant heterozygosity at all loci. The paternal haplotypes inherited in the offspring were different and now the patient and his sibling shared only 3 out of 6 alleles resulting them to be HLA-haploidentical. Case II- A 23 year old female suffering from acute myeloid leukemia was referred for HLA genotyping with her sibling. The sibling was HLA-identical and HSCT was performed. After two years, the graft failed and the patient required second transplant. Low resolution testing of parents indicated sharing of one HLA haplotype. This was confirmed by high resolution sequence based typing at all 6 loci. It was observed that the maternally inherited haplotype was the one shared between the parents therefore, the father appeared to be HLA-identical with the patient and could serve as second donor. Conclusion: High resolution HLA typing is advantageous in select cases for identifying suitable donor.\n\n",
"Meenakshi Singh, Selma Zenia D’Silva, Pankaj Diwedi, Sachin Punatar, Anant Gokarn, Avinash Bonda, Manisha Tambe, S.B. Rajadhyaksha, and Navin Khattry\n\nTata Memorial Centre, Mumbai\n\nIntroduction: Leukemic patients without a full HLA matched donor have an option for haploidentical, matched unrelated donor or unrelated cord transplantation. Recently, Killer Immunoglobulin like receptor (KIR) genes which modulate Natural Killer cell alloreactivity has been linked with predicting transplant outcomes.\n\nStudy Design and Methods: We present two haploidentical hematopoietic stem cell transplants (HHSCT), wherein after engraftment there was graft loss and auto recovery of host cells in one of the patient and relapse in second. We hypothesized that this phenomenon was due to KIR ligand matches/mismatches. The KIR ligand matches/mismatches were examined in both the GvH (graft versus host) and HvG (Host versus graft) direction.\n\nResults: Patient 1 and 2 were 11/12 and 12/12 match for HLA A, B, C, DRB1, DQB1 and DPB1 in GvH respectively, and their donors were heterozygous 6/12 and 7/12 HLA match in HvG directions. It was observed that in both patients, there was KIR activating receptor (aKIR) match and stronger KIR inhibitory receptor (iKIR) mismatch in GvH direction. This resulted in removal of leukemic cells. Moreover, in the HvG direction, the stronger aKIR match led graft rejection.\n\nConclusion: This study highlights the implication of KIR-HLA interaction in predicting graft survival.\n\n",
    "Jyoti Rajak1, Meenakshi Singh1, Manisha Tambe1, Selma D’Silva1, Shalaka Kadam1, S.B. Rajadhyaksha1 and Navin Khattry2\n\n1HLA &amp; Immunogenetics laboratory, Department of Transfusion Medicine, Tata Memorial Hospital 2Bone Marrow Transplantation Unit, ACTREC\n\nIntroduction: In the absence of identical matched sibling donor for hematopoietic stem cell transplantation, haploidentical relatives have become an alternative treatment for patient with hematologic malignancies. The rates of graft failure in patient with anti-HLA antibodies have been reported to be in between 24-83%, with higher rates in haploidentical transplantation recipient. The haploidentical donor’s HLA specificities may vary in immunogenicity from those with potential to cause graft rejection to those that are non immunogenic and have been designated as permissible (acceptable) mismatches in haploidentical transplant.\n\nMethod: In the present study the 34 patients who underwent haploidentical/mismatched SCT were evaluated for HLA Class I, II antibodies pre transplant and post transplant using the Luminex assays.\n\nResults: On analysis of Anti- HLA Class I and II antibodies of patients it was determined that nine were positive for Class I and eighteen for Class II antibodies. The HLA Class II antibodies had an occurrence of 52% in patients leading to acute graft versus host disease (AGVHD) and graft failure (GF) in 22% and 16 % respectively in patients who had engrafted post transplantation.\n\nConclusion: Our studies suggest that anti - HLA antibodies may have an adverse impact on transplant outcomes.\n\n",
    "Akanksha Sharma, Gaurav Sharma, Shekhar Neolia, Dipendra Kumar Mitra, Uma Kanga\n\nDepartment of Transplant Immunology and Immunogenetics, All India Institute of Medical Sciences, New Delhi\n\nIntroduction: The donor chimerism monitoring following Hematopoietic Stem Cell Transplant (HSCT) is essential for evaluating graft outcome. Quantitative evaluation was based on Short Tandem Repeats (microsatellites) variability. However, the population specific genetic propensity of STRs in context of graft outcome informativeness is largely unknown in North Indians.\n\nMaterial &amp; Methods: A total of 164 patients (57.3% malignant, 42.7% non-malignant diseases) who underwent HSCT at AIIMS, New Delhi, were followed regularly (2016-2018). Molecular chimerism was monitored using multiplex STR-PCR based AmpFlSTRIdentifiler Kit (Applied Biosystems) which included 15 STRs located on chromosomes 2, 3, 4, 5, 7, 8, 11, 12, 13, 16, 18, 19, 21, X and Y. The fragment analysis was performed on 3130xl genetic analyzer using Gene Mapper software. Percentage informativity was calculated for each STR locus differing between patient and their respective donors.\n\nResults: Informativity was highly variable among the analyzed STRs. The informativity ranged from 52-74%. Percent informative index of D18S51 was observed to be the highest while; D5S818 and D16S539 were less informative. A new set of STRs (including 2 new markers, ACTBP2 and ApoIII) are under evaluation for possible replacement of AmpFISTR kit.\n\nConclusion: Here we report the informativeness of these STRs based on observed heterozygosity. We need to elucidate whether the in-use STRs are best option for Indian population or an alternate set of markers specific for our population should be identified for post-transplant monitoring. The trends observed in both malignant and non-malignant cohorts are being assessed and will be presented.\n\n",
    "Shashi Kapoor Gautam, Ajeet Alwadhi, Mohd. Abid, Akanksha Sharma, Sunil Kumar, Uma Kanga\n\nDepartment of Transplant Immunology and Immunogenetics, All India Institute of Medical Sciences, New Delhi\n\nIntroduction: Thalassemia poses a significant burden on health status in India, with a high prevalence rate. Hematopoietic stem cell transplantation (HSCT) is the most commonly used curative therapy, but, has limitation. Only 30-35% of patients who find a HLA matched sibling are benefited from this therapy. As an alternate approach to find matched donor, parents and CVS samples were tested to benefit the remaining 65% patients who were unable to undergo HSCT due to lack of suitable donors.\n\nMaterial and Methods: A total number of 176 thalassemic patients and their 352 family members were referred by Haematology AIIMS, New Delhi for identifying a potential donor. HLA typing for HLA-A, B &amp; DRB1 was performed using rSSOPluminex technology (Immucor/One Lambda); any ambiguity observed was resolved using Sequence Based Typing (GENDX). In addition to this, Chorionic Villus Samples (CVS) from pregnant mothers obtained for thalassemia carrier trait screening, were also submitted to our department to examine the probability of HLA match with patient.\n\nResults: Majority of the patients were males (74%) with a mean age of 4.8 years. 27% of the patients (48/176) were able to find HLA-identical donor. In addition, 25% of the CVS samples (3/12) were also observed to be HLA-identical with their thalassemic siblings. Parental HLA typing was performed in 9 families where consanguinity was observed. This helped in identifying parent as a donor for 2 additional cases (22%).\n\nConclusion: We conclude that the CVS and parental typing can help expand the donor pool for thalassemic patients.\n\n",
    "Manisha Tambe, Jyoti Rajak, Shalaka Kadam, Ashok Patel, Selma D’Silva, Sunil B. Rajadhyaksha, Meenakshi Singh\n\nTata Memorial Hospital Parel Mumbai\n\nIntroduction- The quality assurance program is essential in order to ensure uniform and efficient testing protocol. It is achieved by monitoring the three phases pre-analytical (sample collection), analytical (HLA Typing) and post-analytical procedures (interpretation and issue of reports).\n\nMethods: The HLA lab, TMH performs molecular typing for HLA class I and II by PCR SSP/ SSO/SBT method to screen donors for patients undergoing Bone marrow transplantation. The HLA typing procedure involves sample collection, sample processing, interpretation and entering of results. The three parameters i.e. pre-analytical (involves sample collection, mislabeling, sample integrity, sample volume, shipping, requisition lacking information), analytical (consisting of accessioning problem, sample mix-up, lab accident, technical. error, Interpretation Error, External and Internal proficiency testing) and post-analytical (consisting of reporting error and turnaround time) procedures are monitored every month.\n\nResults: The current study represents data of 1895 individuals who underwent HLA typing from December 2016 to August 2018. On monitoring all the parameters, the following percentages were observed for each of the mentioned parameter. Pre-Analytical: Collection Problem (0.42%), Shipping Problem (0.25%), Requisition with required information (1.25%). Analytical: Technical Error (0.16%), Interpretation Error (0.25%), External Proficiency (98.86%), Internal Proficiency (85.85%) Post- Analytical: Turnaround Time (1.01%)\n\nConclusion: Monitoring of the parameters indicated that, the parameters were within the set threshold values required for an accredited HLA laboratory.\n\n"]

var sponsor_adds_sov = [UIImage(named: "immucor"),UIImage(named: "jeenomics"),UIImage(named: "one lambda"),UIImage(named: "dr-mahatme"),
                        UIImage(named: "speaker_3"),UIImage(named: "jeenomics"),UIImage(named: "one lambda"),UIImage(named: "premos"),
                        UIImage(named: "immucor"),UIImage(named: "jeenomics"),UIImage(named: "one lambda"),UIImage(named: "premos"),
                        UIImage(named: "immucor"),UIImage(named: "jeenomics"),UIImage(named: "one lambda"),UIImage(named: "premos"),
                        UIImage(named: "immucor"),UIImage(named: "jeenomics"),UIImage(named: "one lambda"),UIImage(named: "premos"),
                        UIImage(named: "immucor"),UIImage(named: "jeenomics"),UIImage(named: "one lambda"),UIImage(named: "premos"),
                        UIImage(named: "immucor"),UIImage(named: "jeenomics"),UIImage(named: "one lambda"),UIImage(named: "premos"),
                        UIImage(named: "immucor"),UIImage(named: "jeenomics"),UIImage(named: "one lambda"),UIImage(named: "premos"),
                        UIImage(named: "immucor"),UIImage(named: "jeenomics"),UIImage(named: "one lambda"),UIImage(named: "premos"),
                        UIImage(named: "immucor"),UIImage(named: "jeenomics"),UIImage(named: "one lambda"),UIImage(named: "premos"),
                        UIImage(named: "immucor"),UIImage(named: "jeenomics"),UIImage(named: "one lambda"),UIImage(named: "premos"),
                        UIImage(named: "immucor"),UIImage(named: "jeenomics"),UIImage(named: "one lambda"),UIImage(named: "premos"),
                        UIImage(named: "immucor"),UIImage(named: "jeenomics"),UIImage(named: "one lambda"),UIImage(named: "premos")]

class SearchViewController: UIViewController {
    
    
    var searchedCountry = [String]()
    var searching = false
    @IBOutlet weak var countrySearch: UISearchBar!
    @IBOutlet weak var tblView: UITableView!
    override func viewDidLoad() {
        super.viewDidLoad()
        countrySearch.delegate = self
        // Do any additional setup after loading the view, typically from a nib.
    }
    
    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }
}

extension SearchViewController: UITableViewDelegate, UITableViewDataSource {
    
    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
        if searching {
            return searchedCountry.count
        } else {
            return abstract_titles.count
        }
    }
    
    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
        let cell = tableView.dequeueReusableCell(withIdentifier: "cell")
        if searching {
            cell?.textLabel?.text = searchedCountry[indexPath.row]
        } else {
            cell?.textLabel!.numberOfLines = 2
            cell?.textLabel!.lineBreakMode = .byWordWrapping
            cell?.textLabel?.text = abstract_titles[indexPath.row]
        }
        return cell!
    }
    
    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
        indexSovNumber = indexPath.section
        mySovIndex = indexPath.row
        
        if mySovIndex == 0 || mySovIndex == 1 || mySovIndex == 3 || mySovIndex == 4 ||
            mySovIndex == 5 || mySovIndex == 6 || mySovIndex == 7 || mySovIndex == 8 {
            performSegue(withIdentifier: "orgcomseg", sender: self)
        }
        if mySovIndex != 1 && mySovIndex != 0 && mySovIndex != 2 && mySovIndex != 9 && mySovIndex != 3 && mySovIndex != 4 &&
            mySovIndex != 5 && mySovIndex != 6 && mySovIndex != 7 && mySovIndex != 8 {
            performSegue(withIdentifier: "esegue", sender: self)
        }
        
    }
}

extension SearchViewController: UISearchBarDelegate {
    
    func searchBar(_ searchBar: UISearchBar, textDidChange searchText: String) {
        searchedCountry = abstract_titles.filter({$0.lowercased().prefix(searchText.count) == searchText.lowercased()})
        searching = true
        tblView.reloadData()
    }
    
    func searchBarCancelButtonClicked(_ searchBar: UISearchBar) {
        searching = false
        searchBar.text = ""
        tblView.reloadData()
    }
    
}
